web/forms.py

```
@@ -1677,3 +1677,18 @@ class Meta:
         model = StudyGroup
         # You might exclude fields that are set automatically.
         fields = ["name", "description", "course", "max_members", "is_private"]
+
+
+class NFTBadgeForm(forms.ModelForm):
+    class Meta:
+        model = Achievement
+        fields = ["title", "description", "achievement_type", "badge_icon", "course"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)